BookingsApi/BookingsApi.Domain/Hearing.cs

```diff
@@ -215,6 +215,12 @@ public void RemoveParticipant(Participant participant)
             {
                 endpoint.AssignDefenceAdvocate(null);
             }
+
+            foreach (var linkedParticipant in participant.LinkedParticipants.ToList())
```

Comment:
Can achieve the same in a single line
`participant.LinkedParticipants.Clear();`